### PR TITLE
Small Fixes

### PR DESCRIPTION
**Pull request by janosg**
- Restrict pandas to >=0.24 and <1.0 in meta.yaml and other files. Otherwise conda build fails. See #143 
- Add enhancement issue template from respy

### CODE REVIEW

File: meta.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ requirements:
     - python {{ python }}
     - joblib
     - numpy>=1.16.0
-    - pandas>=0.24
+    - pandas<=0.25
```
Comment (tobiasraabe) [inline meta.yaml]: Same for environment.yml and tox? Actually, it would be great if you can fix the issues with pandas v1 very quickly.

Comment (janosg) [reply meta.yaml]: Done, thanks. I'll try to do it soon (or even better, find someone who wants to do it).

Comment (hmgaudecker) [reply meta.yaml]: Nit-picky here, but don't you want
```
- pandas>=0.24,<1.0
```
here to be on the safe side? https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

Comment (janosg) [reply meta.yaml]: Yes. Just didn't know that this works ;-)

**Pull request merged**